OtpCore/OtpAuthUri.cs

```diff
@@ -25,12 +28,13 @@ public class OtpAuthUri
                 Issuer = issuer;
                 Label = $"{Issuer}:{Account}";
                 uriString =
-                    $"otpauth://{Type.ToString().ToLower()}/{Label}?secret={Secret}&issuer={Issuer}&algorithm={Utilities.OtpHmacAlgorithmToString(Algorithm)}";
+                    $"otpauth://{Type.ToString().ToLower()}/{Uri.EscapeDataString(Label)}?secret={Secret}&issuer={Issuer}&algorithm={Utilities.OtpHmacAlgorithmToString(Algorithm)}";
```

Comment:
You still need to perform `Uri.EscapeDataString(Issuer)` for the `issuer` query string parameter value. So:
```
uriString =
                    $"otpauth://{Type.ToString().ToLower()}/{Uri.EscapeDataString(Label)}?secret={Secret}&issuer={Uri.EscapeDataString(Issuer)}&algorithm={Utilities.OtpHmacAlgorithmToString(Algorithm)}";
```